src/QMCDrivers/QMCDriverInput.h

```diff
@@ -78,7 +78,7 @@ class QMCDriverInput
   input::PeriodStride config_dump_period_;
   IndexType starting_step_              = 0;
   IndexType num_crowds_                 = 0;
-  IndexType walkers_per_rank_           = 0;
```

Comment:
This must be set to an invalid input option since it should come only from input or calculation.  We have talked about dropping it from the input spec(i.e. just set total_walkers = your_desired_walkers_per_rank * ranks).  Since the repored max_num_threads can be complete nonsense depending on the threading framework, including on Summit if the hardware threads are visible to openmp, I don't want to automatically set total_walkers = max_threads. Input validation should be added instead.

I think the ambiguous 'walkers' should have no place in the input spec. By all means we can start a section of the manual for the "New" input. But I favor breaking immediately with the old input spec when its ambiguous or nested in such a way to create XML node ownership ambiguity and the old defaults should be eliminated when they don't support the batched concurrency we're working toward which will most performant with the most common size of system.